### DISCUSSION TITLE
Resource creation latency for kafka connect and connector by strimzi

### DISCUSSION THREAD

**Discussion by yli-brex** [strimzi/strimzi-kafka-operator#6594] [Q&A] [answered]
Hey,

I'm wonder if strimzi has latency metric for create Kafka related resources for example kafka connect and connector. :) 

I didn't find related document and metrics from strimzi doc, sorry if I missed something.

Thanks,
Yujie

**Comment by scholzj** [comment, marked as answer]
There is a reconciliation time metric, which describes how long the reconciliations are taking. But the operator in general does not distinguish between _brand new deployment_ and just _checking/updating existing deployment_ (not just in the metric but in the code in general). So not sure sure it is exactly what you are looking for, but there is probably nothing else.
